deps/triemap/triemap.c

```diff
@@ -174,9 +175,10 @@ int TrieMapNode_Add(TrieMapNode **np, const char *str, tm_len_t len, void *value
     // if it was deleted, make sure it's not now
     n->flags &= ~TM_NODE_DELETED;
     *np = n;
-    // if the node existed - we return 0, otherwise return 1 as it's a new node
-    rv += term && !deleted ? 0 : 1;
-    return rv;
+    // it is an existing node, the size is the same but cardinality increases
+    *existing = 1;
```

Comment:
Quite weird to always set `existing` to `1`. Makes it seem redundant, can you explain?